### PR TITLE
Math Magicians: Components

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done
- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called ```components```.
- Inside ```components```, created a new ```Calculator.js``` file.
- In ```Calculator.js```, created a React component that matches the required design.

### CODE REVIEW

**Review by Whoistolu — COMMENTED** [review]
Hi @rmiyoyo 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the Live Demo section from the README since this project is yet to be deployed and deployment is not yet a requirement. Also, fix the MIT link as the License is currently broken. License agreements allow a licensee to use a product that the licensor already owns.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.css
Comment (cynthiainga) [inline src/App.css:1]: - [ ] As mentioned in the requirements the `Calculator.js` component should match the given design 🙃 

Your design

Screenshot 2023-09-17 at 00 30 51

Given design

Screenshot 2023-09-17 at 00 31 17

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (cynthiainga) [inline src/index.js:17]: - [ ] As mentioned in the project requirements all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) should be deleted which means all the files and images in the public directory except the `index.html` file, the index.css, the reportWebVitals.js and setupTests.js files, App.test.js, and the logo.svg 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @rmiyoyo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] The project has a directory called `components` ✔️ 
- [ ] There is a `Calculator.js` file inside the components directory ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] No linters errors ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:
- I think it would be nice if you add your own logo to your README file or if you don't have a logo kindly delete that part 😁 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**